<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1643</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTextBrowser{
 font-family: Consolas,monaco,monospace;
}

QTextEdit {
 font-family: Consolas,monaco,monospace;
}

QCheckBox{
 height: 20px;
}

#Simulate_SimulatorTextBrowser{
 font-size: 19px;
}</string>
  </property>
  <widget class="QTabWidget" name="Application_TabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1621</width>
     <height>861</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Simulate_Tab">
    <attribute name="title">
     <string>Simulate</string>
    </attribute>
    <widget class="QTextBrowser" name="Simulate_RegistersTextBrowser">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>271</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Simulate_SimulatorTextBrowser">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>60</y>
       <width>1111</width>
       <height>741</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas','monaco','monospace'; font-size:19px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="Simulate_ButtonVerticalLayout">
      <item>
       <widget class="QPushButton" name="Simulate_LoadButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Simulate_ReloadButton">
        <property name="text">
         <string>Reload File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Simulate_RunButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Simulate_StepButton">
        <property name="text">
         <string>Step Over</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Simulate_RandomizeButton">
        <property name="text">
         <string>Randomize Machine</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="Simulate_ConsoleLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>360</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Console</string>
     </property>
    </widget>
    <widget class="QLabel" name="Simulate_RegistersLabel">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Registers</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Simulate_FileNameTextBrowser">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>1131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Simulator</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="Simulate_ConsoleTextBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>451</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Simulate_ConsoleLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>441</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>IN:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>OUT:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>350</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Jump To:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Simulate_JumpToLineEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>350</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>31</width>
       <height>741</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>20</width>
       <height>721</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="breakpoint1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint10">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint11">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint12">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint13">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint14">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint15">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint16">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint17">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint18">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint19">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint20">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint21">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint22">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint23">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint24">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint25">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint26">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint27">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint28">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint29">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint30">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint31">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint32">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breakpoint33">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Edit_Tab">
    <attribute name="title">
     <string>Edit</string>
    </attribute>
    <widget class="QTextEdit" name="Edit_EditorTextEditor">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>1431</width>
       <height>611</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Edit_FileNameTextBrowser">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>1471</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Edit_ConsoleTextBrowser">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>660</y>
       <width>1471</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>121</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="Edit_ButtonVerticalLayout">
      <item>
       <widget class="QPushButton" name="Edit_NewButton">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Edit_SaveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Edit_LoadButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Edit_AssembleButton">
        <property name="text">
         <string>Assemble</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTextBrowser" name="Edit_LineNumberTextBrowser">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>41</y>
       <width>41</width>
       <height>611</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
